### PR TITLE
Delet all DMI migrated visits in prod

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

Remove DMI migrated data from DB

## What is the intent behind these changes?

To allow us to re migrate all DMI visits in prod

### CODE REVIEW

File: src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                        JOIN legacy_data ld ON ld.visit_id = v.id
+                        JOIN prison p ON p.id = v.prison_id
+                        WHERE p.code = 'DMI'
+    -- Delete
+    DELETE FROM visit WHERE id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
+    DELETE FROM visit_contact WHERE visit_id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
+    DELETE FROM visit_notes WHERE visit_id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
+    DELETE FROM visit_support WHERE visit_id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
+    DELETE FROM visit_visitor WHERE visit_id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
+    DELETE FROM legacy_data WHERE visit_id IN (SELECT visit_id FROM tmp_delete_migrated_visits);
```
Comment (dhirajshettymoj) [inline src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql:22]: should these be in reverse order with visit last?

File: src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    -- Drop temporary tables
+    DROP TABLE  tmp_visits_with_sessions;
+    DROP TABLE  tmp_visits_to_sessions;
+    DROP TABLE  tmp_visits_to_dup_sessions;
+
+COMMIT;
```
Comment (dhirajshettymoj) [inline src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql:29]: we have got END; in the other scripts.

File: src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql (hunk 1 above)
Comment (AledWEvans) [reply src/main/resources/db.scripts/test/R__VB-2248_Delete_DMI_migrated_visits.sql:22]: they are in correct order

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**